core/templates/pages/topic-editor-page/topic-editor-page.module.ts

```diff
@@ -19,6 +19,10 @@
 import 'core-js/es7/reflect';
 import 'zone.js';
 
+import 'third-party-imports/angular-js.import';
+import 'third-party-imports/headroom.import';
+import 'interactions/codemirrorRequires.ts';
```

Comment:
Why is this added?

Reply:
It was there in the `.import.ts` file but it should be here. imo, the `.import.ts` should have jquery and module as the first two things.

Reply:
Makes sense.